app/app.js

```diff
@@ -10,7 +10,7 @@ Ember.MODEL_FACTORY_INJECTIONS = true;
 App = Ember.Application.extend({
   modulePrefix: config.modulePrefix,
   podModulePrefix: config.podModulePrefix,
-  Resolver,
```

Comment:
I think that's my mistake. js-standard doesn't like them, and I'm in that weird half-way stage of implementing some js-standard stuff, and not others.